mcdu.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title>MCDU</title>
-	<meta charset="utf-8"/>
```

Comment:
This is actually the proper xml syntax. Else the tag is never closed.

Reply:
Actually, having something like that is only for XHTML compatibility. This document is HTML5 so there is no need for that.